packages/browser/src/stack-parsers.ts

```diff
@@ -51,13 +51,22 @@ function createFrame(filename: string, func: string, lineno?: number, colno?: nu
 }
 
 // Chromium based browsers: Chrome, Brave, new Opera, new Edge
+const chromeRegexNoFnName = /^\s*at (\S+?)(?::(\d+))(?::(\d+))\s*$/i;
```

Comment:
could we add a comment here for what this would match, as an example? makes it easier to follow this than to try to parse the regex 😅